src/css/cs/items/_kitchensink.scss

```diff
@@ -396,6 +396,22 @@
     background-image: url('/uc_cust/data/portal/images/uc-icon-warning.svg');
   }
 
+  // Visually hidden class that will allow us to provide text that only a screen reader
+  // to provide extra information to accessible users.  This class should be used when
+  // embedding a link within a message catalog entry that displays text containing a link that
+  // opens in a new window
+  // i.e <a href="http://financialaid.berkeley.edu/loans" target="_blank" data-added-new-window="true">Loans <span class="uc-visually-hidden" style="">Opens in new window</span></a>
```

Comment:
@marcpechaitis not a blocker but might be good to split this into multiple lines (as a follow up). Also, can we remove the empty `style` attribute?